release-index.yaml

```diff
@@ -1,6 +1,11 @@
 releases:
-  - rc_name: rc--2024-10-03_01-30
+  - rc_name: rc-2024-10-11-14_35
     versions:
+      - name: base
+        version: 6fb2fd1f0512c81578ea0d1984c48851e6c08b0d
+      - name: overload
```

Comment:
```suggestion
      - name: hyper-overload
```
?

Reply:
That's not the name of the branch.

https://github.com/dfinity/ic/commits/rc-2024-10-11-14_35-overload